virt/getting_started/virt-using-the-cli-tools.adoc

```diff
@@ -24,9 +24,7 @@ include::modules/virt-installing-virtctl-client-yum.adoc[leveloffset=+2]
 
 include::modules/virt-virtctl-commands.adoc[leveloffset=+1]
 
-== Using libguestfs
-
-include::modules/virt-creating-pvc-with-virtctl-guestfs.adoc[leveloffset=+2]
+include::modules/virt-creating-pvc-with-virtctl-guestfs.adoc[leveloffset=+1]
```

Comment:
Optional. Add a comment with the actual title name would make looking the content easier in the preview link as module titles are not as descriptive, IMO.